docs/NotebookInstructions.md

```diff
@@ -14,29 +14,16 @@ This example notebook demonstrates how to use the [Dataset Insights](https://git
 * Create a virtual environment named `synthdet_env` using Conda, and activate it:
 
 ```bash
-conda create -n synthdet_env python=3.7
+conda create -n synthdet_env python=3.8
 conda activate synthdet_env
 ```
 
-* Install Dataset Insights:
+* Install datasetinsights package from PyPI:
 
 ```bash
 pip install datasetinsights
 ```
 
-* Install Jupyter:
-
-```bash
-conda install jupyter
-```
-
-* You now need to create a kernel for your virtual environment, and add your virtual environment to Jupyter:
-
-```bash
-conda install -c anaconda ipykernel
-python -m ipykernel install --user --name=synthdet_env
```

Comment:
For me, without these two commands, synthdet_env does not appear as a selectable environment inside the notebook. Tested on both Windows and OSX. With them, but skipping `conda install jupyter`, things seem to be working.

Reply:
@mkamalza Yeah, I just verified what you said. Somehow `ipykernel` are not installed properly as part of the `pip install datasetinsights` process. I can run the notebook because it tries to use other kernel that already have datasetinsighs installed on my laptop.